packages/pubsub/src/subscription.js

```diff
@@ -716,6 +832,24 @@ Subscription.prototype.setAckDeadline = function(options, callback) {
 };
 
 /**
+ * Create a Snapshot object. See {module:pubsub/subscription#createSnapshot} to
+ * create a topic.
```

Comment:
Gotcha, I just realized the first "Create" refers to the object and the second refers to the remote object. That's totally confusing, and probably my fault... an eternal docs audit is indeed required!